code/game/gamemodes/blob/theblob.dm

```diff
@@ -241,11 +225,6 @@
 	if(overmind)
```

Comment:
The blob/tesla_act proc should use the "energy" flag for the damage it's applying.